Core/Utilities/Glue2.py

```diff
@@ -0,0 +1,181 @@
+""" Module collecting functions dealing with the GLUE2 information schema
+
+:author: A.Sailer
+
+"""
+
+from pprint import pformat
+
+from DIRAC import gLogger
+from DIRAC.Core.Utilities.ReturnValues import S_OK, S_ERROR
```

Comment:
just `from DIRAC`?

Reply:
I copied the import from Grid.py, f234de5d72 , I thought there was a reason for using the actual location for the import, so I kept it

Reply:
I don't remember any specific reason.